bundles/org.eclipse.equinox.p2.repository/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.equinox.p2.repository;singleton:=true
-Bundle-Version: 2.7.100.qualifier
+Bundle-Version: 2.8.0.qualifier
```

Comment:
API tools now assumes no new API here so 2.7.200 seems sufficient.

Reply:
Maybe this assumption will change if/when a default method is implemented...